config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -186,3 +186,26 @@ presubmits:
     annotations:
       testgrid-dashboards: presubmits-test-infra
       testgrid-tab-name: verify-test
+  - name: pull-test-infra-push-gcloud-terraform
+    cluster: test-infra-trusted
```

Comment:
I'm not comfortable putting presubmits in the trusted cluster at all.

I think testing builds will need a different answer.